package.json

```diff
@@ -8,6 +8,7 @@
     "start:build": "npm run build && serve -s build",
     "test": "react-scripts test",
     "test:update": "npm run test -- --watchAll=false --updateSnapshot",
+    "test:coverage": "react-scripts test --coverage --watchAll=false",
```

Comment:
Maybe `npm run test -- --watchAll=false --coverage`?

Reply:
Good idea 👍 also removed  unnecessary **run** from `npm run test` as `npm test` will do the work
Fixed in: 35026310183d723e292cfc2606eb12c3d890c38d